tests/ffi/common/prepare.sh

```diff
@@ -37,6 +37,18 @@ reload_config() {
 }
 
 prepare_images() {
+   # Update container image_copy_tmp_dir if the image is an OStree.
+   # Default location for storing temporary container image content. Can be overridden with the TMPDIR environment
+   # variable. If you specify "storage", then the location of the
+   # container/storage tmp directory will be used.
+   # By default image_copy_tmp_dir="/var/tmp"
+   # This is a work around and it should not be used constantly
+   if [ -d /run/ostree ]; then
+      exec_cmd "mkdir -p /var/qm/tmp.dir"
```

Comment:
Please use test -d /var/qm/tmp.dir
Assume it is already created, it will generate an error

Reply:
Not sure I understand the suggestion. In this code if checks for the /run/ostree existance with -d, if that directory exists, then all other parts of the case implemented, otherwise skips.
Can you explain what would you like to change here?
Test is a command that is equivalent to the [ command. 
I can implement it this way:
if test -d /var/qm/tmp.dir; then
But I don't see much difference here.
However, if I'll simply use `test -d /var/qm/tmp.dir` without the if statement, it will execute the test and return an exit status (0 for true, 1 for false), but it won't execute any code based on the result.

Reply:
no see what happened to the script if you run `mkdir -p <existing dir>`
so you can use test command, `man test`

Reply:
Ack